cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -162,17 +165,39 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
                         "ports" to listOf(dbPort)
                 )
 
+                // attach database dependency to the node service
                 service["image"] = dockerImage ?: "entdocker.software.r3.com/corda-enterprise-java1.8-${it.runtimeVersion().toLowerCase()}"
                 service["depends_on"] = listOf(dbHost)
+
+                // append persistence volume if it is required
+                if (persistent) {
+                    val volume = "$dbHost-volume"
+                    val volumeDir = "$nodeBuildDir/data"
+                    File(volumeDir).mkdirs()
+                    volumes[volume] = mapOf(
+                            "driver" to "local",
+                            "driver_opts" to mapOf(
+                                 "type" to "none",
+                                 "device" to "$volumeDir",
+                                 "o" to "bind"
+                            )
+                    )
+                    database["volumes"] = listOf("$volume:/var/lib/postgresql/data:z")
```

Comment:
_**EEEEEKK!!**_ It says "postgresql"!!!!! :scream: 

What would I do if I wanted to use a persistent BunnyDB instance instead?

Reply:
Removed